config/member.yaml

```diff
@@ -57,7 +57,7 @@ orgs:
         maintainers:
         - Jeffwan
         repos:
-          resolver-dns: write
+          test-infra: write
```

Comment:
the PR title is totally misleading.. Can you change to something reasonable?

Reply:
OK